frontend/src/views/CellGuide/components/CellGuideCard/index.tsx

```diff
@@ -101,20 +108,33 @@ export default function CellGuideCard({
   const { organismsList, organsMap } =
     useOrganAndOrganismFilterListForCelltype(cellTypeId);
 
-  const uniqueOrgans = Array.from(organsMap.keys());
+  const sdsOrganismsList = organismsList.map((organism) => ({
+    name: organism,
+  }));
 
-  const [selectedOrgan, setSelectedOrgan] = useState(ALL_TISSUES);
+  const sdsOrgansList = Array.from(organsMap.keys()).map((organ) => ({
+    name: organ,
+  }));
+
+  const [selectedOrgan, setSelectedOrgan] = useState<DefaultDropdownMenuOption>(
+    {
```

Comment:
to ensure object equality and keeping the code DRY, instead of creating a new object for all tissue as default, let's just pick it from sdsOrgansList like so: `sdsOrgansList.find(organ => organ.name === ALL_TISSUES)`:

```ts
const [selectedOrgan, setSelectedOrgan] = useState<DefaultDropdownMenuOption>(
    sdsOrgansList.find(
      (organ) => organ.name === ALL_TISSUES
    ) as DefaultDropdownMenuOption
  );
```